wondrous-app/components/Common/CommentItem/styles.tsx

```diff
@@ -0,0 +1,122 @@
+import { Typography } from '@mui/material';
+import { SafeImage } from 'components/Common/Image';
+import DefaultUserImage from 'components/Common/Image/DefaultUserImage';
+import styled from 'styled-components';
+import palette from 'theme/palette';
+
+export const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+  border-bottom: 1px solid ${palette.grey87};
+  padding: 12px 0px;
+  ${({ highlight }) =>
+    highlight &&
+    `
+    @keyframes highlightComment {
+    from {
+      background: ${palette.grey250};
+   }
+    to {
+      background: transparent;
+    }
+  }
+    animation-name: highlightComment;
+    animation-duration: 2s;
+    padding: 2px;
+    border-radius: 4px
+  `}
+`;
+
+export const RoleStatusWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 4px;
+`;
+
+export const Role = styled(Typography)`
+  && {
+    background: ${palette.grey99};
+    color: ${palette.blue20};
+    padding: 4px 8px;
+    height: 26px;
+    font-weight: 500;
+    font-size: 14px;
+    border-radius: 4px;
+  }
+`;
+
+export const ProfileCommentContainer = styled.div`
```

Comment:
you don't need to write custom styles, for example you can use `Box` + `display` prop
```
import Box from '@mui/material/Box';

<Box display="flex">
```

Reply:
Updated